src/editors/data/redux/thunkActions/requests.js

```diff
@@ -290,7 +290,6 @@ export const fetchVideoFeatures = ({ ...rest }) => (dispatch, getState) => {
     requestKey: RequestKeys.fetchVideoFeatures,
     promise: api.fetchVideoFeatures({
       studioEndpointUrl: selectors.app.studioEndpointUrl(getState()),
-      learningContextId: selectors.app.learningContextId(getState()),
```

Comment:
What is the purpose of this removal?

Reply:
This is a separate bug. We are requesting the URL with the course ID at the end, but the URL was changed to be course agnostic. This request is 404ing on prod